AdzerkSDK/AdzerkSDK/AdzerkSDK.swift

```diff
@@ -200,23 +238,21 @@ public typealias ADZUserDBUserResponseCallback = (ADZUser?, Error?) -> ()
     @param callback a simple callback block indicating success or failure, along with an optional `NSError`.
     */
     public func postUserProperties(_ networkId: Int, userKey: String, properties: [String : Any], callback: @escaping ADZResponseCallback) {
-        guard let url = URL(string: "\(AdzerkUDBBaseUrl)/\(networkId)/custom?userKey=\(userKey)") else {
+        guard let url = Endpoint.userDB.baseURL().appending(pathComponent: "\(networkId)/custom", queryItems: [
+            URLQueryItem(name: "userKey", value: userKey)
+        ]) else {
             logger.warn("WARNING: Could not build URL with provided params. Network ID: \(networkId), userKey: \(userKey)")
             callback(false, nil)
             return
         }
         
-        var request = URLRequest(url: url)
-        request.allHTTPHeaderFields = [
-            "Content-Type" : "application/json",
```

Comment:
Is the `content-type` header no longer being set?

Reply:
It is, these lines were redundant because it is configured at the session configuration near the top of this file.